packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.component.js

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import Message from '../../Message';
+
+function FieldTemplate(props) {
+	const groupsClassNames = classNames(
+		'form-group',
+		{ 'has-error': !props.isValid },
+	);
+
+	return (
+		<div className={groupsClassNames}>
+			{
+				props.labelBefore &&
+				<label htmlFor={props.id} className="control-label">{props.labelBefore}</label>
+			}
+			{props.children}
+			{
+				props.labelAfter &&
+				<label htmlFor={props.id} className="control-label">{props.labelAfter}</label>
+			}
+			<Message
```

Comment:
Im not sure since Message handle Description or Error that its wise to use Message for description, since on Error it may hide some helpfull info for end user